api-test/src/main/resources/pms/ABISPartnerV3/GetListofPartnersAssociatedwithLoggedinUserABIS/GetListofPartnersAssociatedwithLoggedinUserABISNegTC.yml

```
@@ -0,0 +1,289 @@
+GetListofPartnersAssociatedwithLoggedinUserABISNegTC:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)